### PR TITLE
Added Google maps section in Articles

### PR DESCRIPTION
**Pull request by dev-aritra**
:pencil2: **Section**: Articles

By sending this pull request to contribute I verify that I adhere to the Xamarin Universal Library guidelines:

- [x] The pull request adheres to the repository's `Code of Conduct`.
- [x] I have followed the section's `Contributing Guidelines`.
- [x] I understand, `Titles` should be capitalized. I have checked the grammar.
- [x] I have searched for previous suggestions before making this new one, as mine could be a duplicate.
- [x] I understand, my contribution may be removed at any time if any of these guidelines are broken.
- [x] The submitted contribution is not owned by a company which sole purpose it is to advertise a product of product line.

Only the collaborators of this repository decide whether a contribution is eligible or not. They will determine if the rules are followed and if the quality of the content is high enough. 

Opening another PR, after the first one is closed in a small period of time will not be reevaluated. If a PR is closed, the issues with your contribution are too significant to be fixed within a matter of hours.

### CODE REVIEW

**Review by yauhenipakala — CHANGES_REQUESTED** [review]
Hi, thank you for contributing!

I think `Google Maps` isn't a common section.
Could you please think about the different name of the section?
For example: `Maps`

**Review by yauhenipakala — APPROVED** [review]

**Comment by dev-aritra** [comment]
@wcoder Yeah I was thinking the same, but then thought does anybody uses apple maps/ bing maps these days. What do you say?

**Comment by yauhenipakala** [comment]
@dev-aritra I know about using Carto, Mapbox, MapKit with Xamarin, maybe in future be good articles about these maps.

**Comment by dev-aritra** [comment]
@wcoder Yeah that sounds right. So, for now, shall I just rename the section to be Maps, and later on when we have articles from different categories of maps, then we would create subsections inside the map section?

**Comment by yauhenipakala** [comment]
> @wcoder Yeah that sounds right. So, for now, shall I just rename the section to be Maps, and later on when we have articles from different categories of maps, then we would create subsections inside the map section?

Yes

Also, please fix the title and description of the section.

**Comment by dev-aritra** [comment]
@wcoder cool, will make the changes and comment then. Thanks for the guidance.

**Comment by dev-aritra** [comment]
@wcoder please check now.

File: src/articles/README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Contributions are very welcome! Please read the [contribution guidelines](contri
 - [Artificial Intelligence](article-topics/Artificial-Intelligence.md)
 - [Bluetooth](article-topics/Bluetooth.md)
 - [Features](article-topics/Features.md)
+- [Maps](article-topics/Maps.md)
```
Comment (yauhenipakala) [inline src/articles/README.md]: Regroup by alphabetical

**Review by yauhenipakala — CHANGES_REQUESTED** [review]

File: src/articles/README.md (hunk 1 above)
Comment (dev-aritra) [reply src/articles/README.md]: Totally missed that, changed it.

**Review by yauhenipakala — APPROVED** [review]

**Pull request merged**